jumpscale/packages/stellar_stats/bottle/stellar_stats.py

```diff
@@ -49,4 +49,32 @@ def get_stats():
     return results
 
 
+@app.route("/api/total_tft")
+def total_tft():
+    query_params = request.query.decode()
+    network = query_params.get("network", "public")
+    tokencode = query_params.get("tokencode", "TFT")
+    detailed = j.data.serializers.json.loads(query_params.get("detailed", "false"))
+
+    collector = StatisticsCollector(network)
+    stats = collector.getstatistics(tokencode, detailed)
```

Comment:
this process takes a very long time, maybe we can introduce caching for 10 mins using redis like the get_stats method. if the results are there in redis fine get it directly if no, we can set it